.eslintrc.json

```diff
@@ -56,15 +56,20 @@
         "alphabetize": { "order": "asc" },
         "newlines-between": "always"
       }
-    ]
+    ],
+    "max-lines": ["error", { "max": 500 }],
+    "max-params": ["error", { "max": 4 }],
+    "max-statements": ["error", { "max": 15 }],
+    "complexity": ["error", { "max": 20 }]
```

Comment:
🟢 I hope I would never write code that breaks these rules 😄 ... but even so, I'd treat these as "warning" not "error".

Reply:
I would much rather people add ESLint-ignore in cases they genuinely believe warrant the complexity to unbreak the build, than ignore complex code warnings.